.github/ISSUE_TEMPLATE.md

```diff
@@ -0,0 +1,50 @@
+### Description
+
+Briefly describe the issue. Feel free to cross-reference your report with a
+related GitHub or [JIRA][jira] issue for additional context.
+
+### Environment
+
+What operating system and PHP version are you using? Mention if PHP was
+installed through a third-party bundle (e.g. MAMP, XAMPP).
+
+Include version information for the extension, libmongoc, and libbson. This may
+be collected from `phpinfo()`:
+
+```
+$ php -i | grep -E 'mongodb|libmongoc|libbson'
+```
+
+Before reporting an issue, check that you are using the most recent version of
+the [extension][pecl].
+
+### Test Script
+
+If possible, include a PHP script that reproduces the issue. Clarify whether
+this script can be run from the CLI or if it can only be reproduced through a
+web SAPI.
+
+### Expected and Actual Behavior
+
+How did the script fail to perform the expected task? Feel free to include debug
+and/or `var_dump()` output.
+
+If the observed behavior is an unexpected exception, please include the full
+message and a backtrace (if relevant). If you encountered a segmentation fault,
+please include a [GDB backtrace][gdb].
+
+### Debug Log
+
+If the issue relates to internal driver behavior (e.g. connection issues),
+please include a debug log. This may be generated by setting the
+[mongodb.debug][debug] INI option to "stderr" or a directory (useful for web
+SAPIs):
```

Comment:
I think this needs for more words to clarify how to exactly use it including a complete example.
Maybe that belongs to the link?